### PR TITLE
doc: updates the contributing.md file

### PR DESCRIPTION
**Pull request by chandhuDev**
- updates the development section steps to align with how it is present in the end-to-end tests section in the contributing.md file of the repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,16 +32,14 @@ If you're worried or don’t know where to start, check out our next section exp
...
-9. Go to the `web/crux-ui` directory: `cd web/crux-ui`
-10. Install dependencies `npm ci`
-11. Copy the _env.example_ file as _.env_ `cp .env.example .env`
-12. Start the frontend with `npm run start`
-13. After you navigated to `localhost:8000` (this is the default Traefik port) you will see a Login screen
-14. Register an account with whatever e-mail address you see fit (doesn't have to be valid one)
-15. Navigate to `localhost:4436` where you will find your mail as all outgoing e-mails will land here
-16. Open your e-mail message and using the link inside you can activate your account
-17. Fruitful contributing! 🎬
+8. Start the backend in production mode with `npm run start`
```
Comment (robot9706) [inline CONTRIBUTING.md:35]: There seems to be some confusion here. This is the development section, so we'd like to start the backend in development mode. The command is OK though.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,16 +32,14 @@ If you're worried or don’t know where to start, check out our next section exp
...
+9. Repeat steps 3-5 in the `web/crux-ui` folder in a different terminal
+10. Start the frontend in production mode with `npm run start`
```
Comment (robot9706) [inline CONTRIBUTING.md:37]: Some production mode here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nandor-magyar) [reply CONTRIBUTING.md:35]: There is a command alias for start: `npm start`, run could be omitted.

**Comment by polaroi8d** [comment]
Is there a possibility for you to correct the improvements we mentioned or should we close the pull request? I'm asking because we want to keep our repository clean, and there hasn't been any update on it for a while.

**Comment by chandhuDev** [comment]
sorry for the delay , I have been quite busy with some other work.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,16 +32,14 @@ If you're worried or don’t know where to start, check out our next section exp
...
-9. Go to the `web/crux-ui` directory: `cd web/crux-ui`
-10. Install dependencies `npm ci`
-11. Copy the _env.example_ file as _.env_ `cp .env.example .env`
-12. Start the frontend with `npm run start`
-13. After you navigated to `localhost:8000` (this is the default Traefik port) you will see a Login screen
-14. Register an account with whatever e-mail address you see fit (doesn't have to be valid one)
-15. Navigate to `localhost:4436` where you will find your mail as all outgoing e-mails will land here
-16. Open your e-mail message and using the link inside you can activate your account
-17. Fruitful contributing! 🎬
+8. Start the backend in developmnt mode with `npm start`
```
Comment (polaroi8d) [inline CONTRIBUTING.md:35]: Typo in here: "developmnt"

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -32,16 +32,14 @@ If you're worried or don’t know where to start, check out our next section exp
...
-10. Install dependencies `npm ci`
-11. Copy the _env.example_ file as _.env_ `cp .env.example .env`
-12. Start the frontend with `npm run start`
-13. After you navigated to `localhost:8000` (this is the default Traefik port) you will see a Login screen
-14. Register an account with whatever e-mail address you see fit (doesn't have to be valid one)
-15. Navigate to `localhost:4436` where you will find your mail as all outgoing e-mails will land here
-16. Open your e-mail message and using the link inside you can activate your account
-17. Fruitful contributing! 🎬
+8. Start the backend in development mode with `npm start`
+9. Repeat steps 3-5 in the `web/crux-ui` folder in a different terminal
```
Comment (robot9706) [inline CONTRIBUTING.md:36]: Steps 3-5 does not fully apply to `crux-ui`. There's no `DATABASE_URL` neither `CRUX_AGENT_ADDRESS`, only `DNS_DEFAULT_RESULT_ORDER` applies here. So I'm not really sure if step 9. is correct this way.

For step 9, I'd prefer the original text, but replacing `npm run start` with `npm start` and mentioning the fact that you need a different terminal for the backend and frontend is a nice addition though.

I think the UI only needs renaming the .env file and running it, if you could rephrase it that would be cool.

Comment (chandhuDev) [reply CONTRIBUTING.md]: Yup , I agree.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -32,11 +32,11 @@ If you're worried or don’t know where to start, check out our next section exp
 -   Alternatively you can use your machine's LAN IP.
 
 7. Deploy the database with `npm run prisma:migrate`
-8. Start the backend with `npm run start`
-9. Go to the `web/crux-ui` directory: `cd web/crux-ui`
+8. Start the backend in development mode with `npm start`
+9. Go to the `web/crux-ui` directory : `cd web/crux-ui`
```
Comment (polaroi8d) [inline CONTRIBUTING.md:36]: Remove the whitespace in here please: `directory :`

**Review by polaroi8d — CHANGES_REQUESTED** [review]

**Review by robot9706 — APPROVED** [review]

**Review by polaroi8d — APPROVED** [review]
LGTM

**Pull request merged**